R/skim.R

```diff
@@ -67,8 +67,10 @@ skim.default <-function(.data, ...){
   structure(skimmed, class = c("skim_vector", class(skimmed)) )
 }
 
-#' Print useful summary statistic from a data frame returning the data frame
-#'  without modification
+
+#' Skim a data frame without modification
```

Comment:
I actually think this is confusing grammatically which is why I changed it to the wordier version. Open to suggestions though. "Skim a data frame and return it without modification" maybe.   Right now the "without modification" modifies the "data frame" being skimmed.  What's important is that the original data frame is returned and the skim results are always printed.

Reply:
We can go back, but the first sentence is inappropriately long for a documentation title and gets recopied as a doc description:

https://www.rdocumentation.org/packages/skimr/versions/1.0.1/topics/skim_tee

What makes sense for hitting both goals (i.e. short title and clear description) together?